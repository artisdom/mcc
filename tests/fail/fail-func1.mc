void foo() {}

void bar() {}

void baz() {}

void bar() {} /* Error: duplicate function bar */

int main()
{
  return 0;
}
