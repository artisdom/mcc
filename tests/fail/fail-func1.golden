Error: redeclaration of function bar